simulator/kruxsim/mocks/lcd.py

```diff
@@ -141,31 +146,106 @@ def _is_x_flipped():
         flipped_x = Settings().hardware.display.flipped_x_coordinates
     return flipped_x
 
+def string_width_px(string):
+    standard_width = BOARD_CONFIG["krux"]["display"]["font"][0]
+    ko_width = BOARD_CONFIG["krux"]["display"]["font_wide"][0]
+    string_width = 0
+
+    for c in string:
+        if CHINESE_MIN_CODEPOINT < ord(c) < KOREAN_CODEPOINT_MAX:
```

Comment:
similar suggestion: condition to check supported langs with an inclusive between "<=" instead of "<"